test/fixtures/demo.yaml

```diff
@@ -7,7 +7,7 @@ strains:
     content: https://github.com/trieloff/helix-demo.git#master
     static: https://github.com/trieloff/helix-demo.git/htdocs#master
     code: ssh://git@github.com/trieloff/helix-demo.git#master
-    package: trieloff/adeb2be318b4997c8254862e8815d8c62fe4de49
+    package: mrosier/adeb2be318b4997c8254862e8815d8c62fe4de49
```

Comment:
what is try? the package should not contain the namespace.

@trieloff ?

Reply:
ahhh that's odd. That's actually the only reason it's failing! :D is because the packages are named with the namespace in helix-configs!

Reply:
I see a number of use cases for different namespaces in one config:

- you have different developers in one team, but don't want to share open whisk credentials
- you want to use another users actions as a "theme", similar to what @MarquiseRosier alluded to

Reply:
Nitpicking, but maybe we should switch to a dummy namespace in the fixtures and recordings.